sopel/modules/reddit.py

```diff
@@ -236,9 +236,58 @@ def comment_info(bot, trigger, match):
     bot.say(message)
 
 
+def subreddit_sorting(bot, trigger, s, sorting):
+    if sorting == 'new':
+        submissions = list(s.new())
+    elif sorting == 'top':
+        submissions = list(s.top())
+    elif sorting == 'hot':
+        submissions = list(s.hot())
+    elif sorting == 'controversial':
+        submissions = list(s.controversial())
```

Comment:
```suggestion
        submissions = list(s.controversial(limit=10))
```

Also worth considering how this feature might allow fetching the most controversial posts over a time period other than "all" (the default).